src/main/java/salesforce/ui/pages/newcampaign/NewCampaignLightningPopUp.java

```diff
@@ -31,8 +30,6 @@
 
     @Override
     protected void waitUntilPageObjectIsLoaded() {
-        webDriverWait.until(ExpectedConditions.visibilityOf(campaignNameField));
-        webDriverWait.until(ExpectedConditions.elementToBeClickable(campaignNameField));
```

Comment:
Why did you delete these?

Reply:
because that locator appears, but sometimes that locator is not reconigzed